ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/CommandFactories.java

```diff
@@ -265,6 +268,29 @@ private static LogicalSchema buildSchema(final TableElements tableElements) {
       throw new KsqlException("The statement does not define any columns.");
     }
 
+    tableElements.forEach(e -> {
+      if (e.getName().equalsIgnoreCase(SchemaUtil.ROWTIME_NAME)) {
+        throw new KsqlException("'" + e.getName() + "' is a reserved field name.");
+      }
+
+      final boolean isRowKey = e.getName().toUpperCase().equals(SchemaUtil.ROWKEY_NAME);
+
+      if (e.getNamespace() == Namespace.KEY) {
+        if (!isRowKey) {
+          throw new KsqlException("'" + e.getName() + "' is an invalid KEY field name. "
```

Comment:
Both "field" and "column" seem to be used in this method to represent the same thing (e.g. line 268 uses column, the new code uses field). For consistency I'd suggest choosing one or the other and sticking to that.

Reply:
Yeah, the two are often used interchangeably. However, I think it's cleaner to use `column` when referring to a column in the table and `field` only for fields within a `STRUCT`.

I'll update all to `column` in this instance.